docs/examples/all-ligand-explicit.rst

```diff
@@ -0,0 +1,197 @@
+.. _all-ligand-explicit:
+
+Absolute Binding of Binders and Non-binders to T4 lysozyme L99A in Explicit Solvent with Combinatorial Options
+==============================================================================================================
+
+This example performs absolute binding free energy calculations for a series of small molecules that contain known
+binders and nonbinders to the T4 Lysozyme L99A protein.
+
+We take advantage of three advanced features of YANK in this example (please see the
+:doc:`basic tutorial for the YAML sections and binding free energy calculation <p-xylene-explicit>`
+for learning how to set up basic YANK simulations):
+
+1. YANK's ability to build out small molecules from SMILES strings (through OpenEye Toolkit)
+
+   * Please see the `OpenEye Install Instructions <http://docs.eyesopen.com/toolkits/python/quickstart-python/install.html>`_ for installing these tool kits
+
+2. YANK's ability to run multiple ligands through the same commands
+3. YANK's ability to run ``Combinatorial`` options
+
+   * See :ref:`the section below <yank_example_combo>` or the :ref:`full documentation page <yaml_combinatorial_head>` for more information on ``!Combinatorial``
+
+
+This example resides in ``{PYTHON SOURCE DIR}/share/binding/t4-lysozyme``.
+
+Original source ligands collected by the `Shoichet Lab <http://shoichetlab.compbio.ucsf.edu/take-away.php>`_.
+
+Disclaimer on this Example
+--------------------------
+
+This example runs many simulations due to the large library of binders and non-binders. You may find it helpful to
+change the following lines in the YAML file
+
+.. code-block:: yaml
+
+    binders:
+        select: all
+    non-binders:
+        select: !Combinatorial [...]
+
+to
+
+.. code-block:: yaml
+
+    binders:
+        select: <Integer>
+    non-binders:
+        select: <Integer>
+
+where ``<Integer>`` is a single integer representing a line in the corresponding file.
+
+Examining YAML file
+-------------------
+
+Here we look at the ``all-ligands-explicit.yaml`` file in this example, highlighting the differences between this file and similar
+files in other examples.
+
+Options Header
+^^^^^^^^^^^^^^
+
+There are no new features introduced in this header.
+
+Molecules Header
+^^^^^^^^^^^^^^^^
+
+.. code-block:: yaml
+
+    molecules:
+      t4-lysozyme:
+        filepath: input/receptor.pdbfixer.pdb
+        leap:
+          parameters: oldff/leaprc.ff14SB
+      binders:
+        filepath: input/L99A-binders.csv
+        antechamber:
+          charge_method: bcc
+        select: all
+      non-binders:
+        filepath: input/L99A-non-binders.csv
+        antechamber:
+          charge_method: bcc
+        select: !Combinatorial [4,5,6,7,10,11,12,13,14,15,18,19,28,29,30,31,32,33,34,35,36,37,41,42,43,48,49,50,56,58,59]
```

Comment:
This is a weird subset of ligands, isn't it? Did you just exclude ligands we can't deal with?